demos/shared/logfire/otel-collector-config.yaml

```diff
@@ -0,0 +1,24 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+      http:
+        endpoint: 0.0.0.0:4318
+
+exporters:
+  otlphttp:
+    endpoint: "https://logfire-api.pydantic.dev"
+    headers:
+      Authorization: "${LOGFIRE_API_KEY}"
```

Comment:
How does a developer set this logfire key? Do we take that in the run_demo.sh script?

Reply:
no it is currently setup as an environment variable